app/services/i_scream_service.rb

```diff
@@ -0,0 +1,27 @@
+class IScreamService 
+
+  def conn 
+    Faraday.new('http://localhost:9393')
+  end
+
+  def get_keyid(keyword)
```

Comment:
These are hitting the routes currently setup in our Sinatra Microservice

Reply:
We will need more routes/methods to return a bunch of games based on keywords

Reply:
I'm curious since right now this is hitting the Sinatra microservice via that port. So does that mean that we also have to setup a Heroku app for the microservice as well?